dhis-2/dhis-services/dhis-service-analytics/src/main/java/org/hisp/dhis/analytics/DataQueryParams.java

```diff
@@ -633,9 +634,30 @@ public String getKey()
             .add( "timeField", timeField )
             .add( "orgUnitField", orgUnitField )
             .add( "userOrgUnitType", userOrgUnitType )
+            .add( "dimensionItemKeywords", getDimensionalItemKeywords( filters ) )
```

Comment:
Just checking, do we also need this for dimensions (not just filters)?

Reply:
Good one, added the same logic to dimensions too.